### PR TITLE
Dev fix

### PR DESCRIPTION
**Pull request by rolcho**

### CODE REVIEW

**Review by markkovari — APPROVED** [review]

File: sk0-front/src/app/guards/auth.guard.spec.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    store.dispatch = jest.fn();
+
+    const route: ActivatedRouteSnapshot = {} as any;
+    const state: RouterStateSnapshot = {} as any;
+
+    const result = await TestBed.runInInjectionContext(() =>
+      authGuard(route, state)
+    );
+
+    expect(result).toBe(true);
```
Comment (markkovari) [inline sk0-front/src/app/guards/auth.guard.spec.ts:61]:
```suggestion
    expect(result).toBeTrue();
```
Iy you would like, this works as well.

Comment (rolcho) [reply sk0-front/src/app/guards/auth.guard.spec.ts:61]: `Property 'toBeTrue' does not exist on type 'JestMatchers<boolean | UrlTree | Observable<boolean | UrlTree>>'.ts(2339)`

Comment (rolcho) [reply sk0-front/src/app/guards/auth.guard.spec.ts:61]: .toBeTruthy() / .toBe(true)

Comment (markkovari) [reply sk0-front/src/app/guards/auth.guard.spec.ts:61]: Oh, nevermind, typescript things. 👍

**Review by bendesbeus — APPROVED** [review]

**Pull request merged**